### DISCUSSION TITLE
import --quiet={skip,asis}

### DISCUSSION THREAD

**Discussion by trapd00r** [beetbox/beets#3991] [Ideas]
At times I'm importing things in quiet mode asis, and what I have to (remember to) do each time is to edit the config:

```
quiet_fallback: asis
#quiet_fallback: skip
```
I would like a way to enable this from the command line, maybe by an optional argument to --quiet?

**Comment by jackwilsdon** [comment]
This sounds like a reasonable option to have, but for what it's worth you can kind of achieve this using the [`-c FILE` global flag](https://beets.readthedocs.io/en/stable/reference/cli.html#global-flags) to overlay another configuration on top of your base one:

```YAML
# config.yaml
quiet_fallback: skip # <- this is the default so you could just omit this
```

```YAML
# asis.yaml
quiet_fallback: asis
```

```Text
$ beet -c path/to/asis.yaml import -q ...
```

**Reply by trapd00r** [reply]
Good catch.
